dts/riscv/espressif/esp32c6/esp32c6_lpcore.dtsi

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+#include <freq.h>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/interrupt-controller/esp-esp32c6-intmux.h>
+#include <zephyr/dt-bindings/clock/esp32c6_clock.h>
+#include <dt-bindings/pinctrl/esp32c6-pinctrl.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
```

Comment:
why is this needed?

Reply:
child node

Reply:
why do all root child nodes need address/size? This doesn't seem to make any sense. You already have `soc` node (`simple-bus`) properly set.

Reply:
Sorry! I mistook the lines from the comment.

Reply:
@nordicjm This seems to be quite common (https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/common/nordic/nrf54h20.dtsi#L21) and lots of others `.dts` and `.dtsi` . I am not sure why this is needed, but it does trigger a build warning when removing it.